api/auth/index.ts

```diff
@@ -33,7 +33,7 @@ export async function attach(req: EnhancedRequest, res: express.Response, next:
     } else {
         const check = await sessionChainCheck(req, res, accessToken)
         if (check) {
-            logger.info('Attaching auth')
+            logger.info('Attaching user-profile')
```

Comment:
I would log this at debug level, not info. We don't want to pollute the logs with endless messages that don't actually operationally tell us very much